{
  try {
    runtimeService.startProcessInstanceByKey("uncaughtErrorThrownByJavaDelegateOnCallActivity-parent");
  }
 catch (  ActivitiException e) {
    assertTextPresent("No catching boundary event found for error with errorCode '23', neither in same process nor in parent process",e.getMessage());
  }
}
